components/raftstore-v2/src/operation/command/admin/merge/prepare.rs

```diff
@@ -373,9 +388,36 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
                 }
             }
             if pending_peers.is_empty() {
-                let (ch, _) = CmdResChannel::pair();
-                let req = req.take().unwrap();
-                self.on_admin_command(store_ctx, req, ch);
+                let mailbox = match store_ctx.router.mailbox(region_id) {
```

Comment:
The whole piece from L391 to L419 is very similar to L167 to L192 in components/raftstore-v2/src/operation/command/admin/mod.rs. Wondering if the code can be reused.

Reply:
The refactor is done this way because `commit.rs` uses `start_pre_flush` differently than these two files. To create yet another function on top of `start_pre_flush` seems cumbersome.